vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/main/java/com/vaadin/flow/component/menubar/MenuBar.java

```diff
@@ -514,4 +514,11 @@ public void setTooltipText(MenuItem item, String tooltipText) {
 
         item.getElement().setProperty("tooltip", tooltipText);
     }
+
+    /**
+     * Closes any open submenus.
+     */
+    public void close() {
+        getElement().executeJs("this._subMenu.close()");
```

Comment:
nit: accessing protected API here. Ideally, the API would first be added to the web component and then we'd use the public API here with `this.close()` (btw `this._close()` is already also available).

Reply:
I agree. There's https://github.com/vaadin/web-components/issues/728 but it has been dormant for 3 years. I propose to go with the workaround for now.